css/styles.css

```diff
@@ -1 +1,40 @@
-/* CSS */
\ No newline at end of file
+*, *::before, *::after {
+    box-sizing: border-box;
+}
+
+:root {
+--light-grey--: #e4e4e4;
+--pale-grey--:  #f2f2f2;
+--grey--: #808080;
+--sky-blue--: #87cefa;
```

Comment:
- Looks like we're missing an indentation
- Unnecessary double dash at the end of variable name
- This naming theoretically looks nice, but it might occur a problems when at some point we decide to change the color value (e.g.: from #808080 to #ff0000) so the name does not fit anymore.
Better approach for that is to set variable names more abstract. There are few options we can consider:
  - Use UI to name variable (e.g.: `--border-color`, `--link-color` etc.)
  - Use greek symbols (e.g. `--alpha`, `--beta`, `--gamma` etc.)
  - Use abstract color names (based on http://chir.ag/projects/name-that-color/)

TBH, I mostly use the last option for colors (because this allow us to have the most possibilities). Of couse, initially the name fits to the color and later on when we change it, it doesn't anymore but the name is still abstract so we are not in the situation where `--red: blue;`

Reply:
done